packages/instant/src/components/erc20_asset_amount_input.tsx

```diff
@@ -80,25 +81,36 @@ export class ERC20AssetAmountInput extends React.Component<ERC20AssetAmountInput
         );
     };
     private readonly _renderBackupContent = (): React.ReactNode => {
```

Comment:
At this point, it is feeling weird to describe this as "Backup" content.  Instead I would describe it more like "Select Token" content